### DISCUSSION TITLE
TikTok: Post to View

### DISCUSSION THREAD

**Discussion by githubUser01946** [ReVanced/revanced-discussions#832] [Question]
Tiktok have this new feature under a tab called now, its annoying. You have to post something in order to view it and sometimes you might not want to post.

![image](https://user-images.githubusercontent.com/117453319/208311803-61db7271-88ac-4d05-be4a-4167a04bfbdc.png)

**Comment by oSumAtrIX** [comment]
Everything which makes TikTok annoying is a feature /s

**Reply by githubUser01946** [reply]
Yes indeed but this is more annoying it forces more crap to view from my "friends".

**Reply by oSumAtrIX** [reply]
I believe this will be a server side limitation.

**Reply by githubUser01946** [reply]
You're probably right but fingers crossed 🤞
